extensions/Bookalope/index.html

```diff
@@ -1,103 +1,319 @@
+						<div class="spectrum-Form-itemField">
+							<label class="spectrum-Checkbox" for="input-book-autoclean">
+								<input type="checkbox" name="input-book-autoclean" id="input-book-autoclean"
+								       checked="true"
+								       class="spectrum-Checkbox-input" disabled>
+	                            <span class="spectrum-Checkbox-box">
+					            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark"
+					                 focusable="false" aria-hidden="true">
+					              <use xlink:href="#icon-CheckmarkSmall"/>
+					            </svg>
+				            </span>
+							</label>
+						</div>
+					</div>
+					<div class="spectrum-Form-item">
+						<label for="input-book-highlight-issues"
+						       class="spectrum-Form-itemLabel spectrum-FieldLabel--left for-checkbox">Highlight
+							issues</label>
+						<div class="spectrum-Form-itemField">
+							<label class="spectrum-Checkbox" for="input-book-highlight-issues">
+								<input type="checkbox" name="input-book-highlight-issues"
+								       id="input-book-highlight-issues"
+								       class="spectrum-Checkbox-input" disabled>
+	                            <span class="spectrum-Checkbox-box">
+					            <svg class="spectrum-Icon spectrum-UIIcon-CheckmarkSmall spectrum-Checkbox-checkmark"
+					                 focusable="false" aria-hidden="true">
+					              <use xlink:href="#icon-CheckmarkSmall"/>
+					            </svg>
+				            </span>
+							</label>
+						</div>
+					</div>
+				</div>
+
+
+				<div class="form__nav">
+					<button class="spectrum-Button spectrum-Button--cta" id="button-send">
+						<span class="spectrum-Button-label">Upload and convert</span>
+					</button>
+				</div>
+			</div>
+
+
+			<div id="bookalope-update" class="hidden">
+				<div class="form__nav">
+					<button class="spectrum-Button spectrum-Button--cta" id="button-update">
+						<span class="spectrum-Button-label">Upload from server</span>
+					</button>
+				</div>
+
+				<hr class="spectrum-Rule spectrum-Rule--small"/>
+
+				<div class="form__text">
+					<p class="spectrum-Body4">Open this document in
+						<a href="#" id="a-bookalope" class="spectrum-Link">
+							<svg class="spectrum-Icon spectrum-Icon--sizeXS"
+							     focusable="false" aria-hidden="true">
+								<use xlink:href="#icon-LinkOut"/>
+							</svg>
+							Bookalope
+						</a>
+					</p>
+				</div>
+				<div class="spectrum-Form spectrum-Form--small">
+					<div class="spectrum-Form-item">
+						<label for="input-book-download" class="spectrum-Form-itemLabel spectrum-FieldLabel--left">Download
+							as</label>
+						<div class="spectrum-Form-itemField">
+							<div class="spectrum-FieldGroup spectrum-FieldGroup--horizontal">
+								<select id="input-book-download" name="input-book-download"
+								        class="spectrum-Textfield spectrum-Dropdown-select">
+									<option value="epub" selected>EPUB</option>
```

Comment:
I’ve changed this on the server, should be
```html
<option value="epub3" selected>EPUB</option>
```

Reply:
Done.

Reply:
Commit https://github.com/jenstroeger/Bookalope-CEP/pull/1/commits/eedf2606d2b55bbdc469b0f76671c120b90fd34f.